src/fhir/CarePlan.DCW.CogLevel1.NoDementiaDx.json

```diff
@@ -0,0 +1,75 @@
+{
+    "resourceType": "CarePlan",
+    "id": "TBD",
+    "description": "DCW cognition domain, level 1 (which is only for patients who don't have a dementia Dx)",
+    "status": "active",
+    "intent": "order",
+    "category": [
+        {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "719091000000102",
+                    "display": "Questionnaire"
+                }
+            ],
+            "text": "Questionnaire"
+        }
+    ],
+    "subject": {
+        "reference": "Patient/TBD",
+        "display": "Doe, John TBD"
+    },
+    "activity": [
+        {
+            "detail": {
+                "instantiatesCanonical": [
+                    "Questionnaire/[ID for ecog12]"
+                ],
+                "code": {
+                    "coding": [
+                        {
+                            "system": "http://snomed.info/sct",
+                            "code": "719091000000102",
+                            "display": "Questionnaire"
+                        }
+                    ],
+                    "text": "Everyday Cognition (ECOG) 12 (short form) questionnaire"
+                },
+                "status": "active",
+                "scheduledTiming": {
+                    "repeat": {
+                        "frequency": 1,
+                        "period": 1,
+                        "periodUnit": "d"
+                    }
+                }
```

Comment:
@achen2401  don't implement `scheduledTiming` yet... if and when we do implement the PlanDefinition to CarePlan `$apply` operation, that operation will apply this and other logic, and the CarePlan that it creates will simply indicate what needs to be done at the moment. If we're unable to implement that, then we'll likely use some mix of `scheduledTiming` and other logic.